apps/remote_control/lib/lexical/remote_control/bootstrap.ex

```diff
@@ -82,4 +82,12 @@ defmodule Lexical.RemoteControl.Bootstrap do
       File.cd!(configured_root)
     end
   end
+
+  defp set_up_workspace(%Project{} = project) do
+    :ok = Project.ensure_workspace_exists(project)
+
+    project
+    |> Project.workspace_path(".gitignore")
+    |> File.write("*\n")
+  end
```

Comment:
I think this should go into `project.ex`, as it's an aspect of the project (and this makes it easier to test too). Plus, in the very near future, there will be other stuff in the project directory.

something_like:
```elixir
Project.ensure_workspace(project)
```
```elixir
# project.ex
def ensure_workspace(project) do 
  with :ok <- ensure_workspace_directory(project) do
   ensure_git_ignore(project)
  end
end

def ensure_git_ignore(%Project{} = project) do 
  git_ignore_contents = """
  *
  """
  git_ignore_path = workspace_path(project, ".gitignore")
  if File.exists?(git_ignore_path) do 
    :ok
  else
    File.write(git_ignore_path, git_ignore_contents)
  end
end
```